src/standalone/Vegetation/Canopy.jl

```diff
@@ -427,6 +427,16 @@ function ClimaLSM.make_update_aux(
         K = extinction_coeff(ld, θs)
         PAR .= compute_PAR(RT, canopy.radiation, t)
         NIR .= compute_NIR(RT, canopy.radiation, t)
+        e_sat =
+            Thermodynamics.saturation_vapor_pressure.(
+                Ref(thermo_params),
+                T,
+                Ref(Thermodynamics.Liquid()),
+            )
+        e = FT(2.655 * q * P / (q + 1.6455))
+        rel_hum = e / e_sat
+        day = round(t / 24 / 3600)
```

Comment:
Im not sure this is correct. `t` is measured from some reference that is set elsewhere. if that reference is not January 1 always, this wouldnt give the right answer.


We implicitly define it as the start date of the data here:
https://github.com/CliMA/ClimaLSM.jl/blob/99ccdaa7a1b68b649827f938898398b92e8b5afd/experiments/integrated/ozark/ozark_met_drivers_FLUXNET.jl#L81
since this starts at 0..


and the same time appears here:
https://github.com/CliMA/ClimaLSM.jl/blob/99ccdaa7a1b68b649827f938898398b92e8b5afd/experiments/integrated/ozark/ozark_met_drivers_FLUXNET.jl#L118

so, I think we need to store the reference date somewhere? let me ask around and see what the other models do for this.

Reply:
Hmm good point, I see what you mean

Reply:
There might be a function called julian day or something like that. It gets any date and give back the julian day.

Reply:
When we prescribe albedo (and eventually reanalysis data) from a file (as a function of lat/lon/t), one of the things we need to define is this "SimInfo" object: https://github.com/CliMA/ClimaLSM.jl/blob/ae8985563721f5d44d35b3907e08c077f9c5461d/src/shared_utilities/FileReader.jl#L80

this eventually gets stored in the PrescribedData object which has the info required to drive the simulation with albedo (reanalysis data) as a function of lat/lon/t.

My thought is that what we are doing with the PrescribedAtmosphere and PrescribedRadiativeFluxes is very similar. we use data to drive the system. and we need a reference date.

For now, I think it is OK to assume that when we drive the system with temporal data, we need to define a reference time, and that can get stored in the object that has the prescribed data info. so I'd say let's add a reference date to the prescribed atmos and prescribed RT types.

What do you think @Espeer5 ?

A longer term project for us is to unify the way we prescribe data across climalsm. I think our use cases would be:
- at a site, pass in functions of time or constants
- global runs: use the PrescribedData object that @juliasloan25 has made which will have a static and temporal flavor (or handle both cases neatly under the hood).

Reply:
This makes sense to me, that way any time we are accessing data at `t` we can see what `t` is in reference to. It is a little bit redundant for now to specify it to both the `PrescribedRadiativeFluxes` and `PrescribedAtmosphere` but it makes sense as a way to standardize it now and try to unify the way it is prescribed down the road as you said.